src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Infrastructure/IOCDataAccessModule.cs

```diff
@@ -2,30 +2,32 @@
 {
     using System;
     using System.Linq;
-    using VolleyManagement.Crosscutting.Contracts.Infrastructure.IOC;
-    using VolleyManagement.Data.Contracts;
-    using VolleyManagement.Data.MsSql.Queries;
-    using VolleyManagement.Data.MsSql.Repositories;
-    using VolleyManagement.Domain.ContributorsAggregate;
-    using VolleyManagement.Domain.FeedbackAggregate;
-    using VolleyManagement.Domain.GamesAggregate;
-    using VolleyManagement.Domain.PlayersAggregate;
-    using VolleyManagement.Domain.RequestsAggregate;
-    using VolleyManagement.Domain.RolesAggregate;
-    using VolleyManagement.Domain.TeamsAggregate;
-    using VolleyManagement.Domain.TournamentRequestAggregate;
-    using VolleyManagement.Domain.TournamentsAggregate;
-    using VolleyManagement.Domain.UsersAggregate;
+    using Crosscutting.Contracts.Infrastructure.IOC;
+    using Contracts;
+    using Queries;
+    using Repositories;
+    using Domain.ContributorsAggregate;
+    using Domain.FeedbackAggregate;
+    using Domain.GamesAggregate;
+    using Domain.PlayersAggregate;
+    using Domain.RequestsAggregate;
+    using Domain.RolesAggregate;
+    using Domain.TeamsAggregate;
+    using Domain.TournamentRequestAggregate;
+    using Domain.TournamentsAggregate;
+    using Domain.UsersAggregate;
 
+#pragma warning disable S1200 // IoC registration
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)